Exercises/Google-Page/Index.html

```diff
@@ -3,8 +3,51 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Google Search</title>
+        <link rel="stylesheet" href="Style.css"/>
 </head>
 <body>
-    
+                <nav>
+                    <div>
```

Comment:
Here you have div that doesn't explain what it means. Better to add some class and style as class.
Just as you did with next `<div class="nav-right">`
Just take in consideration for next html pages.